scripts/clean_osm_data.py

```diff
@@ -17,18 +17,18 @@
 logger = logging.getLogger(__name__)
 
 
-def prepare_substation_df(df_all_substations):
+def prepare_substation_df(df_all_buses):
     """
     Prepare raw substations dataframe to the structure compatible with PyPSA-
     Eur.
 
     Parameters
     ----------
-    df_all_substations : dataframe
+    df_all_buses : dataframe
         Raw substations dataframe as downloaded from OpenStreetMap
     """
     # Modify the naming of the DataFrame columns to adapt to the PyPSA-Eur-like format
```

Comment:
In the spirit of the code refactoring you still done in the PR, wouldn't it make sense to replace here `PyPSA-Eur` with `PyPSA-Earth`? :)

Reply:
Done :)